clients/client-python/gravitino/api/model_version_change.py

```diff
@@ -65,6 +65,17 @@ def update_uri(uri: str):
         """
         return ModelVersionChange.UpdateUri(uri)
 
+    @staticmethod
+    def update_aliases(add_aliases, delete_aliases):
```

Comment:
Here you still use "delete" rather than "remove", I think you should unify them all.